<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>537</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>200</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>FileModifier</string>
  </property>
  <property name="windowOpacity">
   <double>1.000000000000000</double>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>636</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <layout class="QVBoxLayout" name="maskLayout" stretch="0,0">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="maskLabel">
          <property name="text">
           <string>Маска входных файлов:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="maskLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="deleteLayout">
        <property name="sizeConstraint">
         <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
        </property>
        <property name="rightMargin">
         <number>66</number>
        </property>
        <item>
         <widget class="QCheckBox" name="deleteInCheckBox">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LayoutDirection::RightToLeft</enum>
          </property>
          <property name="text">
           <string>Удалять входные файлы</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="destDirLayout">
        <item>
         <widget class="QLabel" name="destDirLabel">
          <property name="text">
           <string>Папка назначения:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="destDirComboBox">
          <property name="editable">
           <bool>true</bool>
          </property>
          <property name="currentText">
           <string/>
          </property>
          <property name="placeholderText">
           <string>Выберите путь до папки назначения</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="onExistLayout">
        <item>
         <widget class="QLabel" name="actionOnExistLabel">
          <property name="text">
           <string>Действие при повторении файла:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="actionOnExistComboBox">
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="editable">
           <bool>false</bool>
          </property>
          <property name="currentText">
           <string/>
          </property>
          <property name="currentIndex">
           <number>-1</number>
          </property>
          <property name="maxVisibleItems">
           <number>3</number>
          </property>
          <property name="maxCount">
           <number>3</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="repeatLayout">
        <item>
         <widget class="QLabel" name="needRepeatLabel">
          <property name="text">
           <string>Повторять по таймеру:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="repeatNoRadioButton">
          <property name="text">
           <string>Нет</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="repeatYesRadioButton">
          <property name="text">
           <string>Да</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="timerLayout">
          <item>
           <widget class="QLabel" name="repeatTimerLabel">
            <property name="text">
             <string>Как часто повторять:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="repeatTimerTimeEdit">
            <property name="minimumTime">
             <time>
              <hour>0</hour>
              <minute>0</minute>
              <second>15</second>
             </time>
            </property>
            <property name="currentSection">
             <enum>QDateTimeEdit::Section::SecondSection</enum>
            </property>
            <property name="displayFormat">
             <string>HH:mm:ss</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="operatorLayout">
        <item>
         <widget class="QLabel" name="modifierOperatorLabel">
          <property name="text">
           <string>Оператор для модификации:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="modifierOperatorComboBox"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="modifierlLayout">
        <item>
         <widget class="QLabel" name="modifierValueLabel">
          <property name="text">
           <string>Значение для модификации</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="modifierValueLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Старт</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
